python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -121,7 +126,7 @@ def get_state(self):
         """Returns the state of the runner."""
         return {
             "epoch": self.epoch,
-            "model": self.model.state_dict(),
+            "model": self.model.cpu().state_dict(),
```

Comment:
does this work also on `cpu`?

Reply:
yes, calling this method on a model that is already on cpu does nothing